src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/DefendantRepositoryFacade.java

```diff
@@ -14,6 +14,6 @@ public DefendantRepositoryFacade(DefendantRepository defendantRepository) {
     }
 
     public Optional<DefendantEntity> findFirstByDefendantId(String defendantId) {
-        return defendantRepository.findFirstByDefendantIdOrderByIdDesc(defendantId);
+        return defendantRepository.findFirstByDefendantId(defendantId);
```

Comment:
In most of these cases I wonder whether we ought to be using `findByDefendantId` without the `first`, a) to make it clear that one result only is expected and b) to enforce that at the JPA as well as DB level

Reply:
Agree with @jhevans

Reply:
It is already there in the code clean up ticket